src/songDetails.ts

```diff
@@ -0,0 +1,54 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import type { SongDetails } from './models';
+
+@customElement('song-details')
+export class SongLine extends LitElement {
+  @property({ type: Object }) details: SongDetails = {
+    imageSrc: 'art/imag_2.jpg',
```

Comment:
```suggestion
    imageSrc: '',
```
or something else that will work if no image is provided